henge.go

```diff
@@ -2,56 +2,27 @@ package main
 
 import (
 	"flag"
-	"os"
+	"fmt"
 
-	"github.com/rtnpro/henge/pkg/generate/dockercompose"
-
-	kapi "k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/unversioned"
-	"k8s.io/kubernetes/pkg/kubectl"
-	"k8s.io/kubernetes/pkg/runtime"
+	"github.com/rtnpro/henge/pkg/loaders"
+	"github.com/rtnpro/henge/pkg/transformers"
 )
 
-func convertToVersion(objs []runtime.Object, version string) ([]runtime.Object, error) {
-	ret := []runtime.Object{}
-
-	for _, obj := range objs {
-
-		convertedObject, err := kapi.Scheme.ConvertToVersion(obj, version)
-		if err != nil {
-			return nil, err
-		}
-
-		ret = append(ret, convertedObject)
-	}
-
-	return ret, nil
-}
-
 func main() {
+	provider := flag.String("provider", "openshift", "Target provider")
```

Comment:
Just a tip. 
In following code you need provider as string.
Co you can do this:
```go
var string provider
flag.VarString(&provider,"provider", "openshift", "Target provider"
```
Than you don't have use dereference `*provider` every time you need it as string.